dkist/io/asdf/converters/inversion.py

```diff
@@ -0,0 +1,27 @@
+from ndcube.asdf.converters.ndcollection_converter import NDCollectionConverter
+
+
+class InversionConverter(NDCollectionConverter):
+    tags = ["asdf://dkist.nso.edu/tags/inversion-0.1.0"]
+    types = ["dkist.dataset.l2_dataset.Inversion"]
+
+    def from_yaml_tree(self, node, tag, ctx):
+        from dkist.dataset.inversion import Inversion
+
+        # We are "promoting" the quantities NDCollection object to being the Inversion object
+        quantities = node["quantities"]
+        aligned_axes = tuple(quantities.aligned_axes[key] for key in quantities.keys())
+        return Inversion(
+            node["quantities"],
+            meta=node["meta"],
```

Comment:
We need to handle the `node["quantities"].meta` part of the tree here, probably by adding it to `node["meta"]`